From a6b6280c04078e12eb77351f8e818cb0bc9dfd80 Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Thu, 20 Apr 2017 15:38:00 -0700
Subject: [PATCH] MORAY-411 tools/rsync-to should not rely on moray being
 functional Reviewed by: Cody Peter Mello <cody.mello@joyent.com> Approved by:
 Cody Peter Mello <cody.mello@joyent.com>

---
 tools/rsync-to | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/tools/rsync-to b/tools/rsync-to
index 66d741a..62cbeca 100755
--- a/tools/rsync-to
+++ b/tools/rsync-to
@@ -20,9 +20,7 @@ TOP=$(cd $(dirname $0)/../; pwd)
 NODE=$1
 
 if [[ -z "$MORAY_ZONE" ]]; then
-    MORAY_ZONE=$(ssh $NODE "/opt/smartdc/bin/sdc-vmapi /vms" 2>/dev/null \
-        | json -H -c 'this.tags && this.tags.smartdc_role === "moray"' \
-            -c 'this.state === "running"' 0.uuid)
+    MORAY_ZONE=$(ssh $NODE "vmadm lookup -1 alias=moray0" 2>/dev/null)
 fi
 echo "MORAY_ZONE: $MORAY_ZONE"
 
-- 
2.21.0

